Day-I-Social-Card/social-card/src/components/CardComponents/Card.css

```diff
@@ -1 +1,33 @@
 /* All styles for your Card Components live here */
+/* .card-container {
+    display: flex;
+    flex-direction: column;
+} */
+
+.banner-container img {
+    display: block;
+    max-width: 630px;
+    max-height: 605px;
+    width: auto;
+    height: auto;
+    margin-left: 80px;
+}
+
+.card-content {
+    height: 6.8rem;
+    width: 70.4%;
+    padding: 1rem;
+    margin-left: 80px;
+    border-bottom: 1px solid #E7EDF0;
+    border-left: 1px solid #E7EDF0;
+    border-right: 1px solid #E7EDF0;
```

Comment:
This is the same as doing `border: 1px solid #e7edf0` and then `border-top: none`